modules/customProtocol/shared/src/main/scala/coursier/cache/protocol/CustomprotocolHandler.scala

```diff
@@ -0,0 +1,19 @@
+package coursier.cache.protocol
+
+import java.io.File
+import java.io.InputStream
+import java.net.{URL, URLConnection, URLStreamHandler, URLStreamHandlerFactory}
+import java.io.FileInputStream
+
+class CustomprotocolHandler extends URLStreamHandlerFactory {
+  def createURLStreamHandler(protocol: String): URLStreamHandler = new URLStreamHandler {
+    protected def openConnection(url: URL): URLConnection = {
+      new URLConnection(url) {
+        def connect(): Unit = ()
+        override def getInputStream(): InputStream = {
+          new FileInputStream(new File(new File(".").getAbsolutePath() + url.getPath()))
```

Comment:
Going from `File` to `String` to `File` again seems odd. Wouldn't the following work?
```suggestion
          new FileInputStream(new File(new File("."), url.getPath.stripPrefix("/")))
```

Reply:
The `stripPrefix` I added may even not be needed.